### DISCUSSION TITLE
When configuring Venus, "bad handshake" appears and the access fails.

### DISCUSSION THREAD

**Discussion by Sela-Hao** [filecoin-project/venus#4477] [Q&A]
When configuring Venus, "bad handshake" appears and the access fails, any one who can tell me how to deal with access failure?

**Comment by Joss-Hua** [comment]
Maybe you need to modify the listen IP in `~ /. venus / config. Toml` to `'0.0.0.0'`, listen to local `'127.0.0.1'` by default, and then restart venus.

**Reply by Sela-Hao** [reply]
OK. I'll try, tks.

**Comment by diwufeiwen** [comment]
> When configuring Venus, "bad handshake" appears and the access fails, any one who can tell me how to deal with access failure?

Can you provide your log and specific operation information?
